### ISSUE TITLE
Enhancement!

### ISSUE THREAD

**Issue by geogerar** [mayuur/MJAlertView#4]
Hello,
A fine tool indeed......but I propose to make an additional enhancement.
When your alert is been displayed you have to disable all background actions...buttons etc just like the original alert view does!

Thanks

George Gerardis

**Comment by mayuur** [comment]
Yup... I was going to provide this in an update... Coming soon :)

Thanks!
